packages/loga/loga.0.0.5/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+synopsis: "Logging library for OCaml"
+maintainer: ["yutopp@gmail.com"]
+authors: ["yutopp"]
+license: "Boost License Version 1.0"
+homepage: "https://github.com/yutopp/loga"
+bug-reports: "https://github.com/yutopp/loga/issues"
+depends: [
+  "ocaml" {build & >= "4.06.0"}
+  "dune" {build & >= "2.5"}
```

Comment:
The build tags should not be there, for dune this has also to do with forward compatibility issues.

Reply:
Thank you for your review! I'll fix it.

Reply:
Thanks!